packages/design-system/src/components/Dropdown/Dropdown.tsx

```diff
@@ -248,7 +248,12 @@ export const Dropdown: React.FC<DropdownProps> = (props: DropdownProps) => {
     ...useLabelProps({
       ...props,
       id,
-      labelClassName: classNames('ds-c-label', 'ds-c-dropdown__label', props.labelClassName),
+      labelClassName: classNames(
+        'ds-c-label',
+        'ds-c-dropdown__label',
+        props.inversed && 'ds-c-label--inverse',
```

Comment:
It looks like this was missing prior to https://github.com/CMSgov/design-system/pull/3099, but I see [the inverse story working on prod](https://design.cms.gov/storybook/?path=/story/components-dropdown--inverse-option). What happened?